libs/angular-components/community/components/form/file-dropzone/file-dropzone.component.ts

```diff
@@ -0,0 +1,352 @@
+  /**
+   * An array of default files that are preloaded in the upload list.
+   * @default false
+   **/
+  defaultFiles = input<File[] | FileDropzone[]>([]);
+  /**
+   * The unique identifier for the input element that this label is associated with.
+   * This ID should match the input element's `id` attribute to ensure accessibility.
+   */
+  id = input<string>();
+  /**
+   * The name attribute for the file input, used for form submission and identifying the field.
+   */
+  name = input<string>();
+  /**
+   * The text label displayed for the file dropzone, providing context for users.
+   */
+  label = input<string>();
+  /**
+   * Additional CSS class names to apply to the dropzone for custom styling
+   */
+  className = input<string>();
+  /**
+   * Specifies how to handle file name conflicts when multiple file of the same name are added.
+   * Options are:
+   * - "append": Adds new files to the end of the list, keeping existing files
+   * - "replace": Replaces existing files with new files of the same name
+   *  @default "append"
+   */
+  mode = input<FileInputMode>("append");
+
+  /**
+   * If true, allows uploading folders instead of just files.
+   * This enables the user to select a folder and upload all its contents.
+   * Default file browser behaviour will prevent upload of files in this state.
+   * @default false
+   */
+  uploadFolder = input<boolean>(false);
+  /**
+   * Validation functions that can be used to validate files.
+   * Each function should return a string error message if validation fails, or undefined if it passes
+   */
+  validators = input<DropzoneValidatorFunction[]>([
+    validateFileSize,
+    validateFileType,
+  ]);
+  /**
+   * Provides helper text or feedback (such as an error or instruction message) to guide the user.
+   */
+  uploadError = model<string>();
+  /**
+   * Disables the file dropzone, preventing user interaction.
+   * @default false
+   */
+  disabled = model<boolean>(false);
+  /**
+   * Provides helper text or feedback (such as an error or instruction message) to guide the user.
+   * By default this is automatically generated based on the `accept` and `maxSize` inputs.
+   */
+  helperText = model<FeedbackTextProps>();
+
+  /**
+   * Callback function triggered when files are added or changed.
+   **/
+  @Output() fileChange = new EventEmitter<FileDropzone[]>();
+  /**
+   * Callback function triggered when a file is removed.
+   **/
+  @Output() fileDelete = new EventEmitter<FileDropzone>();
+
+  fileInputElement =
+    viewChild.required<ElementRef<HTMLInputElement>>("fileInput");
+
+  formatBytes = (bytes: number): string => formatBytes(bytes);
+
+  private _uploadState = () => this._fileService?.uploadState() || "none";
+
+  private _onChange = (_: FileDropzone[]) => {};
+  private _onTouched = () => {};
+
+  private _translationService = inject(TediTranslationService);
+  private _fileService = inject(FileService);
+
+  onDragOverText = signal<string>("");
```

Comment:
is the onDragOverText actually necessary signal? as i understand its only real value is "file-upload.drag-and-drop" and is set in sync with isDragActive, so maybe the isDragActive property alone is sufficient enough?